### ISSUE TITLE
Support Android Select to Speech in Message Input Box

### ISSUE THREAD

**Issue by tashijayla** [simplex-chat/simplex-chat#5622]
In the Simplex Chat Messaging app, the Android Select to Speech accessibility feature does not work in the message input box. When selecting text in the input field, the system does not read the content aloud.

Steps to Reproduce:

Install https://play.google.com/store/apps/details?id=com.google.android.marvin.talkback

And install and setup
https://play.google.com/store/apps/details?id=com.google.android.tts

Open Simplex Chat on an Android device.

Type a message in the input box.

Open Simplex Chat on an Android device.

Type a message in the input box.

Select the text and try to use the Select to Speech feature.

Expected Behavior:
The selected text should be read aloud by Android’s Select to Speech feature.

Actual Behavior:
The feature does not read the selected text.

Proposed Solution:
Enable support for Android’s Select to Speech in the chat input box to improve accessibility.

**Comment by Sam8688** [comment]
Any updates on fixing this?.
Select to speak in chat input box still doesn't work on current version

Version 6.5-beta.1

chat.simplex.app
versionCode 327
